RateLimiter.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"time"
+
+	"github.com/gliderlabs/ssh"
+	"golang.org/x/time/rate"
```

Comment:
I don't want us to introduce this 3rd party dependency, when we have everything we need to implement the rate limiter on our own with a few more lines of code, and leveraging redis, which we already have

Reply:
agree with you, I will work on it